### ISSUE TITLE
[Enhancement] Use a different name for implicit class

### ISSUE THREAD

**Issue by inakianduaga** [t3hnar/scala-bcrypt#20]
## Problem

When importing 

```scala
import com.github.t3hnar.bcrypt._
```

extension method collides with local `Password` class in case one exists. 

## Solution

Obviously the import can be aliased to something else like 

```scala
import com.github.t3hnar.bcrypt.{Password => PasswordExtension}
```

However, since it's probably a very common usecase to have a `Password` class locally, and the name of the implicit extension is completely irrelevant, I suggest to change the name of the implicit class extension

```scala
package com.github.t3hnar

package object bcrypt {

  implicit class StringBcrypt(val pswrd: String) extends AnyVal {
    ...
  }
}
```

**Comment by t3hnar** [comment]
@inakianduaga agree, will release two versions, with deprecation and then with major version bump and cleanup

**Comment by t3hnar** [comment]
Fixed in https://github.com/t3hnar/scala-bcrypt/commit/010aaa46c3a1878cf803dc09736135ac487957ce

released under `v4.0`

**Comment by inakianduaga** [comment]
Well that was ridiculously fast, thanks for updating it!
